token/js/client/token.js

```diff
@@ -427,6 +432,76 @@ export class Token {
     return newAccount.publicKey;
   }
 
+  /**
+   * Create an initialize a new account on the special native token mint.
+   *
+   * In order to wrap native tokens, the account must have a balance of native tokens
+   * when it is initialized with the token program.
+   *
+   * This function sends lamports to the new account before initializing it.
+   *
+   * @param connection A solana web3 connection
+   * @param programId The token program ID
+   * @param owner The owner of the new token account
+   * @param payer The source of the lamports to initialize, and payer of the initialization fees.
+   * @param amount The amount of lamports to wrap
+   * @return {Promise<PublicKey>} The new token account
+   */
+  static async createWrappedNativeAccount(
```

Comment:
I'm curious why you went the `static` route instead of adding a method?

Reply:
It can only be run on the "special case" native token mint, not on any token mint, like createAccount can.

To elaborate, it would be weird to do something like.
```
const srmToken = new Token(connection, serumPubkey, programId, payer)
srmToken.createWrappedNativeAcccount(owner, amount)
```